deps/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+set(ENABLE_PROGRAMS OFF CACHE BOOL "" FORCE)
+set(ENABLE_TESTING OFF CACHE BOOL "" FORCE)
+add_subdirectory(mbedtls)
+set_target_properties(mbedtls PROPERTIES
+        INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_BINARY_DIR}/mbedtls/include)
+
+set(BUILD_TESTING OFF CACHE BOOL "" FORCE)
+set(ENABLE_PROGRAMS OFF CACHE BOOL "" FORCE)
```

Comment:
ENABLE_PROGRAMS duplicate?

Reply:
this was just copied from top-level CMakeLists.txt